packages/@sanity/base/src/presence/PresenceTooltip.tsx

```diff
@@ -34,7 +32,7 @@ export function PresenceTooltip(props: PresenceTooltipProps) {
 
   return (
     <Tooltip content={content} placement={placement}>
-      {children}
+      {children as any}
```

Comment:
Can we remove the `any` cast here if we change the type of `children` to React.ReactNode?